taxcalc/policy_current_law.json

```diff
@@ -180,6 +180,14 @@
             {
                 "year": 2019,
                 "value": 132900.0
+            },
+            {
+                "year": 2020,
+                "value": 137700.0
+            },
+            {
+                "year": 2021,
```

Comment:
Is there a known 2022 value for this parameter?

Reply:
no there is not a known 2022 value of SS_Earnings_c

Reply:
@bodiyang - can you add the 2022 value you found ($147,000) here?  This will mean all expected results files will need to change.